src/screen/MotionDetailScreen.tsx

```diff
@@ -51,19 +51,35 @@ export function MotionDetailScreen(props: PropTypes) {
 
   const {data: motion} = useMotionDetail(params);
 
+  const {data} = useCouncilMembers();
+  const membersCount = data?.members.length ?? 0;
+  const {isCloseable, isVoteable, hasFailed, hasPassed} = useVotingStatus(motion?.votes, membersCount, 'council');
```

Comment:
How about getting the status from this hook and remove all the logic down below?

```suggestion
  const {status} = useVotingStatus(motion?.votes, membersCount, 'council');
```

Reply:
8e333d1